packages/distributor/solana/generated/instructions/closeClaim.ts

```diff
@@ -1,5 +1,4 @@
 import { TransactionInstruction, PublicKey, AccountMeta } from "@solana/web3.js"; // eslint-disable-line @typescript-eslint/no-unused-vars
-import BN from "bn.js"; // eslint-disable-line @typescript-eslint/no-unused-vars
```

Comment:
is it manual?

Reply:
Yes, is there a reason why it's there?

Reply:
the generating template puts it everywhere